test/shell/t1.sh

```diff
@@ -9,7 +9,13 @@
 set -o errexit
 set -o pipefail
 
-timeout --kill-after=3s 1s ./mgmt --help # hello world!
+if [[ $(uname) == "Darwin" ]] ; then
```

Comment:
why don't we put this code in the util.sh script and then we get it everywhere for free?
Then we could actually have it be a $TIMEOUT (in caps) to be used everywhere.